tests/base_new_frmwrk.py

```diff
@@ -165,3 +176,16 @@ def setUpClass(cls,logging="Ensuring environment variables are sourced."):
     @staticmethod
     def is_insight(stream):
         return stream.startswith('ads_insights')
+
+    def expected_page_size(self, stream=None):
+        """
+        return a dictionary with key of table name
+        and value as an integer for the page size of the API requests for that stream.
+        """
+        page_size = {
+            table: properties[BaseCase.API_LIMIT]
+            for table, properties in self.expected_metadata().items()
+            if properties.get(BaseCase.API_LIMIT)}
+        if not stream:
+            return page_size
+        return page_size[stream]
```

Comment:
I'm pretty sure this exists in tap-tester base and doesn't need to be replicated here.  In any case it belongs there as API Limit or page size is on every SaaS tap.  Since it isn't defined everywhere I believe we had to account for the case where the API LIMIT isn't set.

Reply:
Updated base case and removed this over-ride